data/zilliqa/assets/zil14t82ughznxl7r3an329r6uswmc3rpr2dk5c6dm/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Anacas Iron",
+  "symbol": "ANCS",
+  "trusted": true,
+  "config": {
+    "dark": true
+  },
+  "gen": {
```

Comment:
remove this

Reply:
Sorry about  being a dumbass. I'm new to Zilliqa. It's an exciting project